### PR TITLE
Changed upperbound of resolveVal to 2560 (GCOM-1171)

### PR DESCRIPTION
**Pull request by Jessevdpoel**

### CODE REVIEW

File: packages/next-ui/Styles/responsiveVal.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@
 export function responsiveVal(
   min: number,
   max: number,
-  maxBreakpoint = 1280,
```
Comment (mikekeehnen) [inline packages/next-ui/Styles/responsiveVal.tsx:9]: Kunnen we bij deze waarde geen `theme.breakpoints.values.xl` gebruiken?

Comment (paales) [reply packages/next-ui/Styles/responsiveVal.tsx:9]: responsiveVal doesn't have access to the theme object, so that isn't possible without modifications.

Comment (mikekeehnen) [reply packages/next-ui/Styles/responsiveVal.tsx:9]: It would prevent future bugs that look like this, but might be a bit too much for a simple fix.

Comment (paales) [reply packages/next-ui/Styles/responsiveVal.tsx:9]: I can discuss this later how to properly solve it, but for now I'd like to k eep it at 2560

**Pull request closed without merging**